src/types.ts

```diff
@@ -244,6 +244,11 @@ export interface ViewState {
   visibleWidth?: number;
   rowsPerPage?: number;
   page?: number;
+  isMultiPage?: boolean;
+  maxLineCount?: number;
```

Comment:
Is the intention that the user will be able to set the number of lines when printing and/or taking a snapshot?

Reply:
Yes, in future, when we support different row heights we will send probably 3 as maxLineCount instead of 1 for the first version of multi page pdf.